languages/elixir/exercises/concept/conditionals/.meta/design.md

```diff
@@ -0,0 +1,47 @@
+# Design
+
+## Goal
+
+The goal of this exercise to teach students how to use conditional control statements
+
+## Learning objectives
+
+After completing this exercise, the student should:
+
+- Know of the existence of the `cond` function and how to use it.
+- Know how to perform boolean logic using the `===/2` operator function.
+- Know how to return values from a function with conditional logic.
+- Know of the existence of the [atom][atom] type
+
+## Out of scope
+
+- Pattern matching.
+- Function overloading.
+- `case` function.
```

Comment:
case _statements_ I think would be better nomenclature

Reply:
I will update to match the getting started guide `control flow structure`

Reply:
I think one point I am trying to get across to student is that everything is a function, even if it doesn't look like it on the surface.